blocks/library/embed/index.js

```diff
@@ -221,14 +234,21 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 		},
 
 		save( { attributes } ) {
-			const { url, caption, align } = attributes;
+			const { url, caption, align, type, providerNameSlug } = attributes;
 
 			if ( ! url ) {
 				return;
 			}
 
+			const embedClassName = classnames( {
+				[ `wp-block-embed` ]: true,
```

Comment:
Plain strings should use single quotes - this is the Travis error you're seeing.